TLE in unverified-user-message.hbs

**Describe the bug**
We get a Template Linting Error in `unverified-user-message.hbs` because an action is being assigned to an icon. This is causing a test failure - 
```sh
open-event-frontend/templates/components/unverified-user-message.hbs
  4:26  error  Interaction added to non-interactive element  invalid-interactive

===== 1 Template Linting Error
```

**To Reproduce**
Steps to reproduce the behavior:
1. Run `ember test`

**Expected behavior**
There shouldn't be an error.

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04
 - Browser Chrome
 - Version [e.g. 22]

--- a/app/controllers/events/view/edit/sessions-speakers.js
+++ b/app/controllers/events/view/edit/sessions-speakers.js
@@ -57,10 +57,11 @@ export default Controller.extend({
             .then(() => {
               this.get('notify').success(this.get('l10n').t('Your event has been saved'));
               this.set('isLoading', false);
-              if (direction === 'forwards')
+              if (direction === 'forwards') {
                 this.transitionToRoute('events.view.index', data.id);
-              else if (direction === 'backwards')
+              } else if (direction === 'backwards') {
                 this.transitionToRoute('events.view.edit.sponsors', data.id);
+              }
             }, function() {
               this.get('notify').error(this.get('l10n').t('Event data did not save. Please try again'));
             });

--- a/app/controllers/events/view/edit/sponsors.js
+++ b/app/controllers/events/view/edit/sponsors.js
@@ -41,10 +41,11 @@ export default Controller.extend({
             .then(() => {
               this.set('isLoading', false);
               this.get('notify').success(this.get('l10n').t('Your event has been saved'));
-              if (direction === 'forwards')
+              if (direction === 'forwards') {
                 this.transitionToRoute('events.view.edit.sessions-speakers', data.id);
-              else if (direction === 'backwards')
+              } else if (direction === 'backwards') {
                 this.transitionToRoute('events.view.edit.basic-details', data.id);
+              }
             }, function() {
               this.get('notify').error(this.get('l10n').t('Sponsors data did not save. Please try again'));
             });

--- a/app/templates/components/unverified-user-message.hbs
+++ b/app/templates/components/unverified-user-message.hbs
@@ -1,6 +1,6 @@
 {{#if shouldShowMessage}}
   <div class="ui icon error message eight wide column center aligned">
-    <i class="close icon" onclick={{action (mut isMessageVisible) false}}></i>
+    <i class="close icon" onclick={{action (mut isMessageVisible) false}} role="button"></i>
     <i class="info icon"></i>
     <div class="content">
       <div class="header">

